sdk/analytics/src/test/java/com/klaviyo/analytics/model/ModelTests.kt

```diff
@@ -79,21 +79,44 @@ internal class ModelTests : BaseTest() {
         assertEquals("custom", profileMap["custom"])
     }
 
+    @Test
```

Comment:
I think this behavior is good enough™️ , but an alternative would be to make all these named properties _actual properties_ rather than being backed by the overload dict. Then we'd have to override the the `toMap` function as well. Thats a considerably bigger change to the public API though.